src/service/standard-actions-impl.js

```diff
@@ -103,6 +105,23 @@ export class StandardActions {
     );
   }
 
+  /**
+   * Handles initiliazing the theme mode.
+   *
+   * This methode needs to be called on page load to set the `amp-dark-mode`
+   * class on the body if the user prefers the dark mode.
+   */
+  initThemeMode() {
+    const themeMode = this.ampdoc.win.localStorage.getItem('amp-dark-mode');
```

Comment:
Local storage access can throw in private mode, causing AMP initialization to fail (the throw will reach the root of the call graph). This needs to be wrapped in a try-catch.

Reply:
Thanks, @jridgewell I have wrapped the localStorage API calls within a try-catch block